spring-cloud-dataflow-tasklauncher/spring-cloud-dataflow-tasklauncher-function/pom.xml

```diff
@@ -0,0 +1,34 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+		 xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+		 xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+	<modelVersion>4.0.0</modelVersion>
+
+	<parent>
+		<groupId>org.springframework.cloud</groupId>
+		<artifactId>spring-cloud-dataflow-tasklauncher</artifactId>
+		<version>2.9.0-SNAPSHOT</version>
+	</parent>
+
+	<artifactId>spring-cloud-dataflow-tasklauncher-function</artifactId>
+
+	<dependencies>
+
+		<dependency>
+			<groupId>org.springframework.cloud</groupId>
+			<artifactId>spring-cloud-dataflow-rest-client</artifactId>
+			<version>${project.version}</version>
+			<exclusions>
```

Comment:
Kinda interesting use case we never thought of what comes for pom deps and how things are used. If client is used for a limited subset of api's, when things actually work or do not work as there's always potential runtime errors for missing classes. I'm wondering if it's worth an effort to modify dataflow so that you would not need to exclude(I assume something broke?)

Reply:
The tests still work without the exclusions, so I'm not sure what broke. I'll take it out for now and do some more runtime testing.